css/css.css

```diff
@@ -71,3 +71,12 @@ div {
   -webkit-transform: scale(1.3) translateZ(0);
   transform: scale(1.3) translateZ(0);
 }
+
+.codeblock {
+  font-family: monospace
+  padding:2px 4px;
```

Comment:
I guess it’s good practice to add a space after the colon for readability (also below).